kpatch-build/create-diff-object.c

```diff
@@ -1609,18 +1597,11 @@ static void kpatch_replace_sections_syms(struct kpatch_elf *kelf)
 					  relasec->name,
 					  rela->sym->name, rela->addend,
 					  sym->name, rela->addend - start);
-				found = true;
 
 				rela->sym = sym;
 				rela->addend -= start;
 				break;
 			}
-
-			if (!found && !is_string_literal_section(rela->sym->sec) &&
-			    strncmp(rela->sym->name, ".rodata", 7)) {
-				ERROR("%s+0x%x: can't find replacement symbol for %s+%ld reference",
-				      relasec->base->name, rela->offset, rela->sym->name, rela->addend);
-			}
```

Comment:
If this error is getting removed, there needs to be more justification than "we think it is just ok".

Reply:
Also, why does .rela.orc_unwind_ip reference .orc_unwind_ip itself?  That sounds like a different format compared to the x86 implementation, which only references text addresses.

Reply:
> If this error is getting removed, there needs to be more justification than "we think it is just ok".
OK,Let's leave it not changed.

Reply:
> Also, why does .rela.orc_unwind_ip reference .orc_unwind_ip itself? That sounds like a different format compared to the x86 implementation, which only references text addresses.
Yes,the rela format is a little different.